sonar-text-dotnet/pipeline_scripts/analysis_begin_step.ps1

```diff
@@ -1,3 +1,11 @@
+$versionFilePath = "$env:CIRRUS_WORKING_DIR\sonar-text-dotnet\Directory.Build.props"
```

Comment:
NITPICK: Why don`t we use project dir and then directly access the build props in it?